.github/workflows/release-from-main.yml

```
@@ -86,21 +111,34 @@ jobs:
               echo "Renamed $file to $(dirname $file)/$NEW_FILE_NAME"
           done
           echo "renamed_pom_files=$(IFS=","; echo "${RENAMED_FILES[*]}")" >> $GITHUB_OUTPUT
+      - name: Upload POM artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: aar-files
```

Assuming should be:
```suggestion
          name: pom-files
```
?

---

Thank you so much for seeing that one! 🙌